src/control/zoom/ZoomControl.cpp

```diff
@@ -84,15 +89,18 @@ void ZoomControl::zoomOneStep(ZoomDirection direction, utl::Point<double> zoomCe
     }
     this->setZoomFitMode(false);
 
+    double multiplier = 1.0 + this->zoomStep;
+    double newZoomStep = (direction == ZOOM_IN) ? multiplier : 1.0 / multiplier;
```

Comment:
Please do not use inline if statements. They are hard to read and therefore error-prone.